internal/idler/user_idler.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/configuration"
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/openshift/client"
+	userstatus "github.com/fabric8-services/fabric8-jenkins-idler/internal/status"
```

Comment:
Why should `user-idler` be tightly coupled with `statusService`?
What does user-idler which is specific to the user have to do with `status-service` which maintains a list of `disabled` users?